### ISSUE TITLE
Choose Hypothesis

### ISSUE THREAD

**Issue by huypham37** [huypham37/AIML-UM-14#28]

**Comment by huypham37** [comment]
"How do different learning parameters (learning rate, batch size, buffer size) affect the computational efficiency and training performance of PPO in the SoccerTwos environment?"

 Dependent Variables (Metrics):
   - Computational Efficiency:
     * Training time (wall clock)
     * CPU utilization
     * Memory usage
     * Training steps per second
